### ISSUE TITLE
On salt and pepper rules

### ISSUE THREAD

**Issue by Bumbadawg** [LukeSmithxyz/based.cooking#464]
> Don't include salt and pepper and other ubiquitous things in the ingredients list.

Maybe you're not aware that there are 3 main types of salt used in cuisine: fine, coarse, flower. Each have their specific usage. 
For instance, you need coarse salt to crust-roast some specific meat pieces. Fine won't do, and you won't use flower salt when cooking (duh).

Same goes for peppers, lots of sorts, like chilis, many types: full, ground, choped. 

I'd go even further, water. For some recipes like an île flottante requires you to prepare iced water in order to achieve the chemistry reaction.

TLDR; base ingredients aren't ubiquitous, their usage can be. Get these "rules" straight please.
